StudCity.Frontend/src/components/Group/GroupCard.jsx

```diff
@@ -1,29 +1,33 @@
 import React from 'react'
+import GridCard from './GridCard'
+import ListCard from './ListCard'
+
+const GroupCard = ({ group, handleClick, viewMode }) => {
+	const stylesContainerCardMode = () => {
+		switch (viewMode) {
+			case "grid":
+				return "duration-200 hover:scale-90 h-60";
+			case "list":
+				return "h-36 mt-5 w-11/12 mx-auto duration-200 hover:scale-105";
+			default: return "";
+		}
+	};
+
+	const renderCardMode = () => {
+		switch (viewMode) {
+			case "grid":
+				return <GridCard group={group} />;
+			case "list":
+				return <ListCard group={group} />;
+			default: return <></>;
```

Comment:
just my preference, you can simply do default: return;
and please add enter for better code readability and consistency